website/src/graphql/dailp/queries.graphql

```diff
@@ -13,67 +13,41 @@ query DocumentsPages {
   }
 }
 
-query AnnotatedDocument($slug: String!) {
-  document(slug: $slug) {
-    id
-    title
-    slug
-    isReference
-    date {
-      year
-    }
-    bookmarkedOn {
-      formattedDate
-    }
-    sources {
-      name
-      link
-    }
-    audioRecording {
-      resourceUrl
-      startTime
-      endTime
-    }
-    translatedPages {
-      image {
-        url
-      }
-    }
-    chapters {
-      path
+# Data about a document needed to render various high level components, such as
+# a DocumentHeader. Contrast with the fields resolved on DocumentContents query.
+fragment DocumentFields on AnnotatedDoc {
```

Comment:
I would love a better name for this. It isn't quite `DocumentMetadata` since that is already a special named object. But it is a subset of fields useful for rendering lots of little components that don't need everything in `DocumentContentsQuery`

Reply:
Agree with @CharlieMcVicker but I think this is an excellent change otherwise!